Lib/csv.py

```diff
@@ -128,6 +128,11 @@ def __next__(self):
         return d
 
 
+class DictReader(TableReader):
```

Comment:
Would a DeprecationWarning be appropriate here? as this ideally would not affect the behavior of DictReader, and would instead just add the "new" functionality of TableReader.